src/main/java/core/basesyntax/Calculator.java

```
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Calculator {
+    public double calculate(double a, double b, char operation) {
+        if (operation == '+') {
+            return a + b;
+        }
+        if (operation == '-') {
+            return a - b;
+        }
+        if (operation == '/' && a != 0 && b != 0) {
+            return a / b;
+        }
+        if (operation == '*' && a != 0 && b != 0) {
+            return a * b;
+        }
+        if (operation == '^') {
+            return a != 0 || b != 0 ? Math.pow(a, b) : a != 0 && b == 0 ? 1.0 : 0;
+        }
+        return 0;
+    }
+}
```

You should throw an Exception if you use unknown character in kind of operation character